content/knowledge_base/glossary.md

```diff
@@ -2,11 +2,29 @@
 
 ## General Blockchain Concepts
 
-### Consensus
+### Blockchain
 
-TODO: add
+This technology is based on a chain of blocks linked to each other.
+Once a new block is created and inserted on the chain, the new block will have a link to the previous block, creating a chain.
+They all include the hash of the previous block except for the first one, which is called the genesis block, its a zero hash value.
 
-### Blockchain
+There are two types of blockchain, permissionless and permissioned.
+The permissionless blockchain, also called public blockchain, is available to everyone, because of that, anyone can create and track down blocks.
+This type of blockchain comes with some limitations, mainly slow performance and bad scaling.
+
+On the other side, the permissioned blockchain, or often referred to as private blockchain, has limited availability, only being accessible to a certain group instead of being publicly available, they are also faster and better to scale.
+On this type of blockchain, there is no anonymity like in a public blockchain.
+
+The core concents of the a blockchain network are:
+
+* Transparency
+    * Every full node in the network has a copy of the whole chain of blocks, this means that every transaction is available to each member, making the transactions traceable. This makes every transaction more transparent and removes the need for intermediaries, allowing the process to be more efficient.
```

Comment:
@Ullaakut I saw that is better to slip phrases in multiple lines to improve readability but how does it work with bullet points?

Reply:
With bullet points you can do the following:

```markdown
* Point 0 
* This is a multiline markdown bullet point.
It contains multiple sentences separated by newlines.
* Point 2
```

* Point 0 
* This is a multiline markdown bullet point.
It contains multiple sentences separated by newlines.
* Point 2

It changes the rendering a bit unfortunately but in most cases that's actually more readable this way.

Reply:
What do you mean? Should I remove it?

Reply:
Remove what? I was just answering about multiline bullet points in markdown and Ceki suggested splitting a sentence into two 🤔